api/src/controllers/record-utils.js

```diff
@@ -196,7 +181,7 @@ const createByForm = (data, { locale }={}) => {
     type: 'sms_message',
     message: data.message,
     form: smsparser.getFormCode(data.message),
-    sent_timestamp: data.sent_timestamp,
+    reported_date: data.sent_timestamp,
```

Comment:
if we re-align this naming we don't have to create an options object further down which is just a copy of the meta block